<launch>
<node pkg="multi_amr_nectec" type="real_robot_straight" name="real_robot_straight" output="screen" >
 <param name ="team_size" value = "4" />
<param name ="v_linear_threshold" value = "0.3" />
<param name ="v_angular_threshold" value = "0.3" />
<param name ="robot_mass" value = "40" />
<param name ="robot_radious" value = "0.5" />
<param name ="robot_inertia" value = "0.77" />
  <param name ="delta_time" value = "0.01" />
</node>
</launch>